codegenerator/cli/templates/static/codegen/src/ink/components/SyncETA.res

```diff
@@ -11,12 +11,16 @@ let isIndexerFullySynced = (chains: array<ChainData.chainData>) => {
 }
 
 let getTotalRemainingBlocks = (chains: array<ChainData.chainData>) => {
-  chains->Array.reduce(0, (accum, {progress, currentBlockHeight, latestFetchedBlockNumber}) => {
+  chains->Array.reduce(0, (accum, {progress, currentBlockHeight, latestFetchedBlockNumber, endBlock}) => {
```

Comment:
Ok so endBlock is only set if they have it in their config? Makes sense.

Reply:
Yes